docs/content/v2/en/options.md

```diff
@@ -14,6 +14,23 @@ category: Guide
 Sets the global `crossorigin` value of the nuxt-speedkit preloads.  
 The default value is the `crossorigin` value from the [Render Configuration](https://nuxtjs.org/docs/2.x/configuration-glossary/configuration-render#crossorigin).
 
+## `optimizePreloads`
+- Type: `Boolean`
+  - Default: `true`
+
+If set, the preloads and scripts will be optimized.
+
+The following nuxt options are set:
```

Comment:
The following NuxtJS settings are made or overwritten in the nuxt.config:

Reply:
Angepasst!